content/applications/hr/payroll/contracts.rst

```diff
@@ -89,6 +89,17 @@ Contract details
 The contract details section allows for the addition and editing of a contract, and the ability to
 send the contract to the employee for approval and signatures.
```

Comment:
As you indicate, you mention that in the 'Contract Details' tab of a contact form, users can send the contract to the employee for approval and signatures... can you be more specific where in this tab such an action can be performed? Thanks

Reply:
I updated htis section- since you must add templates in here, or you cannot create the contract (and will not see the 'send for signature' button either). I updated htis to be more accurate, and specified these are required fields.